[BUG] DGUS_LCD_UI_MKS with DEBUG_DGUSLCD won't compile

### Did you test the latest `bugfix-2.0.x` code?

Yes, and the problem still exists.

### Bug Description

I am currently trying to port the MKS DWIN screen firmware to a slightly different screen, and I hoped debugging would help me figure out why it won't do the thing. However, if I enable `DEBUG_DGUSLCD` in Marlin/src/lcd/extui/dgus/DGUSDisplay.h, it encounters errors:
```
/mnt/DNI/Downloads/Marlin/Marlin-2.0.9.3-test/Marlin/src/lcd/extui/dgus/mks/DGUSScreenHandler.cpp:759:39: error: 'class GCodeQueue' has no member named 'length'
/mnt/DNI/Downloads/Marlin/Marlin-2.0.9.3-test/Marlin/src/core/serial.h:297:75: error: conversion from 'const int16_t {aka const int}' to 'EnsureDouble' is ambiguous
```
And a cascade of others that are seemingly just side effects. I could probably fix this myself but I'm already three or four levels deep in "I need to fix A so I can fix B" so handing this one off would be pretty neat. None of the other DWIN options have such an issue.

Incidentally, debugging is _entirely_ broken in the bugfix branch as of right now, so that's of little help. The offending lines still exist in the relevant file, though.

### Bug Timeline

<= 2.0.9.3

### Expected behavior

Compile with DGUS/DWIN debug output enabled

### Actual behavior

Compile error; references nonexistent class member.

### Steps to Reproduce

1. Download Marlin 2.0.9.3 (current)
2. Edit Configuration.h to uncomment `#define DGUS_LCD_UI_MKS` (no other edits needed to test)
3. Edit Marlin/src/lcd/extui/dgus/DGUSDisplay.h to uncomment `#define DEBUG_DGUSLCD`
4. Attempt to compile in the latest environment of your choice

### Version of Marlin Firmware

2.0.9.3

### Printer model

Longer LK4 Pro

### Electronics

Stock

### Add-ons

_No response_

### Bed Leveling

_No response_

### Your Slicer

_No response_

### Host Software

_No response_

### Additional information & file uploads
[Configuration.zip](https://github.com/MarlinFirmware/Marlin/files/7883188/Configuration.zip)

--- a/Marlin/src/lcd/extui/dgus/fysetc/DGUSScreenHandler.h
+++ b/Marlin/src/lcd/extui/dgus/fysetc/DGUSScreenHandler.h
@@ -206,7 +206,7 @@ public:
   static void DGUSLCD_SendFloatAsIntValueToDisplay(DGUS_VP_Variable &var) {
     if (var.memadr) {
       float f = *(float *)var.memadr;
-      DEBUG_ECHOLNPGM(" >> ", f, 6);
+      DEBUG_ECHOLNPAIR_F(" >> ", f, 6);
       f *= cpow(10, decimals);
       dgusdisplay.WriteVariable(var.VP, (int16_t)f);
     }

--- a/Marlin/src/lcd/extui/dgus/hiprecy/DGUSScreenHandler.h
+++ b/Marlin/src/lcd/extui/dgus/hiprecy/DGUSScreenHandler.h
@@ -206,7 +206,7 @@ public:
   static void DGUSLCD_SendFloatAsIntValueToDisplay(DGUS_VP_Variable &var) {
     if (var.memadr) {
       float f = *(float *)var.memadr;
-      DEBUG_ECHOLNPGM(" >> ", f, 6);
+      DEBUG_ECHOLNPAIR_F(" >> ", f, 6);
       f *= cpow(10, decimals);
       dgusdisplay.WriteVariable(var.VP, (int16_t)f);
     }

--- a/Marlin/src/lcd/extui/dgus/mks/DGUSScreenHandler.cpp
+++ b/Marlin/src/lcd/extui/dgus/mks/DGUSScreenHandler.cpp
@@ -93,7 +93,7 @@ void DGUSScreenHandler::DGUSLCD_SendFanToDisplay(DGUS_VP_Variable &var) {
 
 void DGUSScreenHandler::DGUSLCD_SendBabyStepToDisplay_MKS(DGUS_VP_Variable &var) {
   float value = current_position.z;
-  DEBUG_ECHOLNPGM(" >> ", value, 6);
+  DEBUG_ECHOLNPAIR_F(" >> ", value, 6);
   value *= cpow(10, 2);
   dgusdisplay.WriteVariable(VP_SD_Print_Baby, (uint16_t)value);
 }
@@ -756,7 +756,7 @@ void DGUSScreenHandler::HandleManualMove(DGUS_VP_Variable &var, void *val_ptr) {
   else if (manualMoveStep == 0x02) manualMoveStep =  100;
   else if (manualMoveStep == 0x03) manualMoveStep = 1000;
 
-  DEBUG_ECHOLNPGM("QUEUE LEN:", queue.length);
+  DEBUG_ECHOLNPGM("QUEUE LEN:", queue.ring_buffer.length);
 
   if (!print_job_timer.isPaused() && !queue.ring_buffer.empty())
     return;

--- a/Marlin/src/lcd/extui/dgus/mks/DGUSScreenHandler.h
+++ b/Marlin/src/lcd/extui/dgus/mks/DGUSScreenHandler.h
@@ -273,7 +273,7 @@ public:
   static void DGUSLCD_SendFloatAsIntValueToDisplay(DGUS_VP_Variable &var) {
     if (var.memadr) {
       float f = *(float *)var.memadr;
-      DEBUG_ECHOLNPGM(" >> ", f, 6);
+      DEBUG_ECHOLNPAIR_F(" >> ", f, 6);
       f *= cpow(10, decimals);
       dgusdisplay.WriteVariable(var.VP, (int16_t)f);
     }

--- a/Marlin/src/lcd/extui/dgus/origin/DGUSScreenHandler.h
+++ b/Marlin/src/lcd/extui/dgus/origin/DGUSScreenHandler.h
@@ -206,7 +206,7 @@ public:
   static void DGUSLCD_SendFloatAsIntValueToDisplay(DGUS_VP_Variable &var) {
     if (var.memadr) {
       float f = *(float *)var.memadr;
-      DEBUG_ECHOLNPGM(" >> ", f, 6);
+      DEBUG_ECHOLNPAIR_F(" >> ", f, 6);
       f *= cpow(10, decimals);
       dgusdisplay.WriteVariable(var.VP, (int16_t)f);
     }

